weblate/checks/duplicate.py

```diff
@@ -94,8 +93,4 @@ def get_description(self, check_obj):
         source = unit.source_string
         for target in unit.get_target_plurals():
             duplicate.update(self.check_single(source, target, unit))
-        return mark_safe(
-            "{} {}".format(
-                escape(self.description), escape(", ".join(sorted(duplicate)))
-            )
-        )
+        return format_html("{} {}", self.description, ", ".join(sorted(duplicate)))
```

Comment:
Looks like we can just return a formatted string, since it doesn't contain any HTML?

Reply:
Sounds correct.

Reply:
Actually I think let's keep it, there's practically no performance difference and in the future if someone decides to add HTML, they won't have to rewrite it.